src/callbacks_stage/positivity_shallow_water.jl

```diff
@@ -0,0 +1,87 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+"""
+    PositivityPreservingLimiterShallowWater(; variables)
+
+The limiter is specifically designed for the shallow water equations.
+It is applied to all scalar `variables` in their given order
+using the defined `threshold_limiter` from the [`ShallowWaterEquations1D`](@ref) struct
+or the [`ShallowWaterEquations2D`](@ref) struct to determine the minimal acceptable values.
```

Comment:
```suggestion
using the defined `threshold_limiter` from the [`ShallowWaterEquationsWetDry1D`](@ref) struct
or the [`ShallowWaterEquationsWetDry2D`](@ref) struct to determine the minimal acceptable values.
```

Reply:
Done in 77f44c650f95e4bc7bcbd284ecd2f60f6b9d0609